Formula/cataclysm.rb

```diff
@@ -14,40 +15,51 @@ class Cataclysm < Formula
     sha256 "e7ea748e9dd53bd0ace6c8456c4eb351616dc9c879621b6724ce742ae2b0d4f2" => :yosemite
   end
 
-  option "with-tiles", "Enable tileset support"
-
   needs :cxx11
 
   depends_on "gettext"
-
-  if build.with? "tiles"
-    depends_on "sdl2"
-    depends_on "sdl2_image"
-    depends_on "sdl2_ttf"
-  end
+  depends_on "lua"
+  depends_on "pkg-config"
```

Comment:
`pkg-config` ought to be a `:build` dependency no ?

Reply:
Yup 👍